java/apitest/src/test/java/e2e/registry/registry.feature

```diff
@@ -343,4 +343,139 @@ Feature: Registry api tests
     When method get
     Then status 200
     And assert response[0].address[0].phoneNo.length == 1
-    And assert response[0].address[0].phoneNo[0] == "444"
\ No newline at end of file
+    And assert response[0].address[0].phoneNo[0] == "444"
+
+    Scenario: write a api test, to test the schema not found error
+      #  make keycloak authentication enabled:false in file application.yml
+      #  make serach providerName: ${search_providerName:dev.sunbirdrc.registry.service.ElasticSearchService}
+         in file application.yml
+        * url baseUrl(http://localhost:8081)
+
+       # invite schema which is unavailable/not found
+        Given url baseUrl
+        And path 'api/v1/{entityName}/invite'
+        method post
```

Comment:
did you run these tests?
run them once. looks like there will be some compilation failures